apps/blur/test.cpp

```diff
@@ -14,6 +14,8 @@
 
 double t;
 
+#pragma GCC diagnostic ignored "-Wcast-qual"
```

Comment:
lgtm except for this pragma. Why is it necessary (as opposed to something like Wno-cast-qual in the build files for this app)?